site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -390,6 +390,9 @@ The possible values are:
 +
```

Comment:
Curious why you had to take so many screenshots?

Reply:
@mooreds 
For each image in the documentation that now shows the Application Theme dropdown, I took a new screenshot. I went back and forth on whether to do this or not since the only one that really pertains to applciation theme is the "Add Application".  If you think its unnecessary I can back most of them out.

Reply:
no worries, makes sense!